openassessment/templates/openassessmentblock/staff_area/staff_area.html

```diff
@@ -0,0 +1,171 @@
+                                    <td class="value">{% trans "Course ID" %}</td>
+                                    <td class="value">{{ classifierset.course_id }}</td>
+                                </tr>
+                                <tr>
+                                    <td class="value">{% trans "Item ID" %}</td>
+                                    <td class="value">{{ classifierset.item_id }}</td>
+                                </tr>
+                                </tbody>
+                            </table>
+                        {% else %}
+                            {% trans "No classifiers are available for this problem" %}
+                        {% endif %}
+                    </div>
+
+                    <div class="staff-info__status ui-staff__content__section">
+                        <a aria-role="button" href="" id="schedule_training" class="action--submit"><span class="copy">{% trans "Schedule Example-Based Assessment Training" %}</span></a>
+                        <div id="schedule_training_message"></div>
+                    </div>
+                {% endif %}
+
+                {% if display_reschedule_unfinished_tasks %}
+                    <div class="staff-info__status ui-staff__content__section">
+                        <a aria-role="button" href="" id="reschedule_unfinished_tasks" class="action--submit"><span class="copy">{% trans "Reschedule All Unfinished Example-Based Assessment Grading Tasks" %}</span></a>
+                        <div id="reschedule_unfinished_tasks_message"></div>
+                    </div>
+                {% endif %}
+            </div>
+        </div>
+    </div>
+
+    <div id="openassessment__staff-info" class="wrapper--staff-info wrapper--ui-staff is--hidden">
+        <div class="staff-info ui-staff">
+            <h2 class="staff-info__title ui-staff__title">
+                <i class="ico icon-caret-right"></i>
+                <span class="staff-info__title__copy">{% trans "Course Staff Information" %}</span>
+                <button class="ui-staff_close_button"><span class="sr">{% trans "Close" %}</span> <i class="icon fa fa-close" aria-hidden="true"></i></button>
+            </h2>
+
+            <div class="staff-info__content ui-staff__content">
+
+                <div class="staff-info__summary ui-staff__content__section">
+                    <dl class="submissions--total">
+                        <dt class="label">{% trans "Response total" %}:</dt>
+                        <dd class="value">{{ num_submissions }}</dd>
+                    </dl>
+                </div>
+
+                <div class="staff-info__status ui-staff__content__section">
+                    {% trans "Location" %}: {{ item_id }}
+                </div>
+
+                <div class="staff-info__status ui-staff__content__section">
+                    <table class="staff-info__status__table" summary="{% trans "Where are your learners currently in this problem" %}">
```

Comment:
@catong I forgot to comment on this. Can we make this a separate PR so that I can wrap this one? I've got to head out now anyway, so let's discuss on Monday.